### PR TITLE
Move raw block feature to GA in 1.18

### PR DESCRIPTION
**Pull request by jsafrane**
/assign @msau42

### CODE REVIEW

File: keps/sig-storage/20190130-csi-raw-block-support.md [hunk 1]
```diff
@@ -162,3 +163,5 @@ and we can test with them
 * Separated NodeStage / NodePublish calls.
 * Fixed volume reconstruction after kubelet restart.
 * Stress tests as noted above.
+
+### K8s 1.18: GA
```
Comment (msau42) [inline keps/sig-storage/20190130-csi-raw-block-support.md]: Add a note that 1.18 is the target? Same below

Comment (jeremyrickard) [reply keps/sig-storage/20190130-csi-raw-block-support.md]: Hey @msau42 @jsafrane can you link to which KEP issue#s this relates to so we can better track? 

Thanks!

Comment (jsafrane) [reply keps/sig-storage/20190130-csi-raw-block-support.md]: There is https://github.com/orgs/kubernetes-csi/projects/4, not sure it should be part of the KEP though.

Comment (jsafrane) [reply keps/sig-storage/20190130-csi-raw-block-support.md]: @msau42, not sure I understood, changed to `K8s 1.18: Targeting GA`

Comment (msau42) [reply keps/sig-storage/20190130-csi-raw-block-support.md]: I guess there's a title for this section, but no content under it. Is there anything we should be adding to the section? More test cases/corner cases to fix/test?

Comment (jsafrane) [reply keps/sig-storage/20190130-csi-raw-block-support.md]: Added this:
> * Disruptive testing with block devices in /dev reordered after reboot.
